numba_dppy/tests/test_dpnp_functions.py

```diff
@@ -267,6 +259,585 @@ def f(a):
             self.assertTrue(np.array_equal(expected, got))
 
 
+@unittest.skipUnless(ensure_dpnp(), "test only when dpNP is available")
+class Testdpnp_random_functions(unittest.TestCase):
+    sizes = [None, 9, (2, 5), (3, 2, 4)]
+
+    def test_random_sample(self):
+        @njit
+        def f(size):
+            c = np.random.random_sample(size)
+            return c
+
+        sizes = [9, (2, 5), (3, 2, 4)]
+        with assert_dpnp_implementaion():
+            with dpctl.device_context("opencl:gpu"):
+                for size in sizes:
+                    result = f(size)
+                    _result = result.ravel()
+                    for i in range(_result.size):
+                        self.assertTrue(_result[i] >= 0.0)
+                        self.assertTrue(_result[i] < 1.0)
```

Comment:
Tests for dpNP functions calls without check for results

I think we need to check that dpnp function was called. 
We do not have to check resutls.
dpNP has its own tests for these function results.

We need to check that function is present and could be called with different data types.

Reply:
Moved to ticket https://github.com/IntelPython/numba-dppy/issues/212